tests/cucumber/features/login.feature

```diff
@@ -0,0 +1,5 @@
+Feature: Log in with CheckIn A person with an identity can log in with CheckIn
+  Scenario: The login button is present
```

Comment:
I'm curious on how do you plan to test other scenarios here, mainly, testing an actual login with Check-in, is that feasible at all?

Reply:
As long as we have proper credentials with Selenium you can do whatever you want. We could easily create a google account for a test user and used it access discourse.

Reply:
Exactly. Discussed this a few times with @nliampotis